algorithms/unit_tests/TestStdAlgorithmsAdjacentFind.cpp

```diff
@@ -315,6 +315,12 @@ void run_all_scenarios() {
 }
 
 TEST(std_algorithms_nonmod_seq_ops, adjacent_find) {
+#if defined(KOKKOS_ENABLE_CUDA) && \
+    defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC
+  if constexpr (std::is_same_v<exespace, Kokkos::Cuda>) {
+    GTEST_SKIP() << "FIXME please";
```

Comment:
Wrong result? Compiler error? Please comment.

Reply:
Wrong result, the `if constexpr` only guards the skip test, so it is being compiled

Reply:
Updated